oneanddone/tests/functional/tests/test_task_assignment.py

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subjectfrom django import forms to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.home import HomePage
+
+
+class TestAvailableTasks():
+    @pytest.mark.nondestructive
+    def test_assign_tasks(self, base_url, selenium, nonrepeatable_task, new_user):
+        home_page = HomePage(selenium, base_url).open()
+        assert not home_page.is_user_logged_in
```

Comment:
There's no need to check that we're not logged in. This is covered by another test.